src/main/java/com/heroiclabs/nakama/DefaultClient.java

```diff
@@ -30,11 +30,11 @@
 import io.grpc.stub.MetadataUtils;
 import lombok.NonNull;
 import lombok.extern.slf4j.Slf4j;
-import lombok.var;
```

Comment:
@lugehorsam here is where we edited a source file to generate the docs. Could you check if this is ok?